steering-committee/kira_evans.md

```diff
@@ -0,0 +1,5 @@
+---
+title: Kira Evans
+repository: https://github.com/kne42
+avatar: https://twitter.com/kne42/photo
```

Comment:
```suggestion
avatar: https://pbs.twimg.com/profile_images/1295752376003911680/w0-kdJ-p_400x400.jpg
```

The current URL is not a link to an image.

Reply:
ah, good catch!